README.md

```
@@ -1,48 +1,189 @@
+ECCUBE_BASE_URL=https://localhost:8081 npx playwright test tests/web-install.spec.ts
+```
 
-- Client ID
-- Client Secret
-- Authorization Endpoint
-- Token Endpoint
-- UserInfo Endpoint
-- Provider Name
+## ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)